### PR TITLE
Initial scaffolding Data Engine Proxy as a new type of governance server

### PR DESCRIPTION
**Pull request by cmgrote**
Likely to be revised as we further integrate between Data Engine OMAS and an example data engine, but need this initial scaffolding to begin minimal unit testing of code (and revisions from there).

### CODE REVIEW

File: open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        A generic proxy connector for plugging in data engine connectors to sync data engine metadata.
+    </description>
+
+    <artifactId>data-engine-proxy-connector</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-connector-framework</artifactId>
+            <version>${open-metadata.version}</version>
```
Comment (planetf1) [inline open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml]: WIth #22 integrated you can (should) omit the version tags in child poms.
If any NEW dependencies are added, an entry should be made in the dependencyManagement section of the top level pom, with a property also from top level pom

File: open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j.version}</version>
```
Comment (planetf1) [inline open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml]: remove

File: open-metadata-implementation/admin-services/admin-services-server/pom.xml [hunk 3]
```diff
@@ -122,6 +122,13 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>data-engine-proxy-services-server</artifactId>
+            <version>${open-metadata.version}</version>
```
Comment (planetf1) [inline open-metadata-implementation/admin-services/admin-services-server/pom.xml]: version should be removed
Also why are common admin services now dependent on an OMAS server?

File: open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-services-server/pom.xml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        The data engine proxy services server-side implementation.
+    </description>
+
+    <artifactId>data-engine-proxy-services-server</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>data-engine-proxy-connector</artifactId>
+            <version>${open-metadata.version}</version>
```
Comment (planetf1) [inline open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-services-server/pom.xml]: All these versions can be removed

File: open-metadata-implementation/admin-services/admin-services-server/pom.xml (hunk 3 above)
Comment (cmgrote) [reply open-metadata-implementation/admin-services/admin-services-server/pom.xml]: the Data Engine Proxy Services Server is a new type of governance server -- not an OMAS.  So it's akin to Discovery Engine Services, Stewardship Services, etc which are all also dependencies

File: open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml (hunk 2 above)
Comment (cmgrote) [reply open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml]: hmm, I only removed the version -- did you mean remove the entire slf4j dependency?

Comment (planetf1) [reply open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml]: no just the version

File: open-metadata-implementation/admin-services/admin-services-server/pom.xml (hunk 3 above)
Comment (planetf1) [reply open-metadata-implementation/admin-services/admin-services-server/pom.xml]: ah! sure.

**Pull request merged**